iot-e2e-tests/common/src/test/java/tests/integration/com/microsoft/azure/sdk/iot/iothub/telemetry/SendMessagesTests.java

```diff
@@ -104,15 +104,7 @@ public void sendLargestMessages() throws Exception
     {
         this.testInstance.setup();
 
-        if (this.testInstance.protocol == AMQPS_WS)
```

Comment:
No need for a special case here. AMQPS_WS can send the same maximum size messages that the other protocols can.